go/integTest/shared_commands_test.go

```diff
@@ -4039,3 +4039,39 @@ func (suite *GlideTestSuite) TestZPopMax() {
 		assert.IsType(suite.T(), &api.RequestError{}, err)
 	})
 }
+
+func (suite *GlideTestSuite) TestZRem() {
+	suite.runWithDefaultClients(func(client api.BaseClient) {
+		key := uuid.New().String()
+		memberScoreMap := map[string]float64{
+			"one":   1.0,
+			"two":   2.0,
+			"three": 3.0,
+		}
+		res, err := client.ZAdd(key, memberScoreMap)
+		assert.Nil(suite.T(), err)
+		assert.Equal(suite.T(), int64(3), res.Value())
+
+		// no members to remove
+		_, err = client.ZRem(key, []string{})
+		assert.NotNil(suite.T(), err)
+		assert.IsType(suite.T(), &api.RequestError{}, err)
+
+		res, err = client.ZRem(key, []string{"one"})
+		assert.Nil(suite.T(), err)
+		assert.Equal(suite.T(), int64(1), res.Value())
+
+		// TODO: run ZCard there
```

Comment:
Why is this TO DO added here? This TO DO should be ZCARD, right?

Reply:
I will merge this PR today. @prateek-kumar-improving, can you have a chat with Yury after he is back. Thanks